main-app/tests/proxy/test-proxy.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+    'use strict';
+    describe('Testing proxy', function () {
+        var constants,
+            $httpBackend;
+        beforeEach(function () {
+            module('Tombola.NoughtAndCrosses.core');
+            inject(function ($injector) {
+                constants = $injector.get('Proxy');
+                $httpBackend = $injector.get('$httpBackend');
+            });
+        });
+
+        it('Ensures makeGame function creates a new game successfully from api', function () {
+            $httpBackend.expectPOST('http://eutaveg-01.tombola.emea:35000/api/v1.0/newgame', {'player1': 'human',
+                'player2': 'human'}).respond({'outcome':'Continue','gameboard':'100000000','winner':0});
+
+            var returnedPromise = constants.makeGame("human", "human");
+            returnedPromise.then(function(response){
+                response.should.deep.equal({'outcome':'Continue','gameboard':'100000000','winner':0});
```

Comment:
This object and the POST response object are the same: you don't want to change one, then the other. 
Store it in a separated variable and deep equal that instead.